tech/languages/csharp/dotnet-installation.md

```diff
@@ -8,26 +8,31 @@ order: 3
 
 ## Fedora .NET SIG repository
 
-.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as containing prebuilt binaries or bundled dependencies.) Please be patient, we should have reasonable packages within a few months.
+.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as using prebuilt tools to build the source) but are open source. Please be patient, we should have reasonable packages within a few months.
 
 [Fedora .NET SIG copr](https://copr.fedorainfracloud.org/coprs/g/dotnet-sig/dotnet)
 
 To enable the .NET SIG's copr repository:
 
 `$ sudo dnf copr enable @dotnet-sig/dotnet`
 
-## .NET Core SDK
+### .NET Core SDK
 
 .NET Core SDK will enable building and publishing C# source code.
 
 `$ sudo dnf install dotnet-sdk-2.0`
 
-## .NET Core Runtime only
+### .NET Core Runtime only
 
 You can install runtime only, if you are just deploying already prebuilt application.
 
 `$ sudo dnf install dotnet-runtime-2.0`
 
+## Microsoft .NET Core packages
+
+[.NET Core packages built by Microsoft](https://www.microsoft.com/net/download/linux) contain some proprietary libraries of the .NET Framework.
```

Comment:
So it can never be in Fedora proper?

Reply:
Microsoft built sh..stuff is proprietary, always been, always will be - applies to VSCode as well as dotnet. Source code is open source, but their built binaries are proprietary and can not be re-distributed. Our builds are _free_